src/components/Dropzone/Dropzone.tsx

```diff
@@ -276,6 +279,7 @@ export const Dropzone = ({
           <MobileUI>
             {variant === DropzoneVariant.light ? (
               <Button
+                type="button"
```

Comment:
We should probably have a look at this button in general, but wouldn't make more sense to still make it possible to pass a type to this? I don't think we're using it as a submit button if other external people use a submit button it's breaking for them, right?

Reply:
So I think it would be nice to take that into account OR add a breaking change in the commit message.

Reply:
This button is always used to open the file manager from within the `Dropzone`, so I don't think anyone would be using it to submit a form. Nonetheless I have added a breaking commit.